primitives/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 0.101.0 - 2024-11-15
+
+This is the first "real" release of the `primitives` crate, as such it
```

Comment:
In 53d3ab18b9d3385c8c72fe422a36235c825f44c3:

I won't hold the PR up over it but "as such" makes this a run-on sentence.

Reply:
> The run-on sentence is undoubtedly the most common grammar mistake made by writers with otherwise sound writing skills.

But I don't feel too bad about it.

ref: https://www.grammar-monster.com/glossary/run-on_sentence.htm